packages/core/src/routes/experience/index.ts

```diff
@@ -45,17 +46,72 @@ export default function experienceApiRoutes<T extends AnonymousRouter>(
       koaExperienceInteraction(tenant)
     );
 
+  /**
+   * @api {post} /experience Create a new interaction
+   */
+  router.put(
```

Comment:
i suggest remove this typo comment and, if we need to document apis, use swagger

Reply:
removed